services/migrations/restore_github_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"context"
+	"testing"
+
+	"code.gitea.io/gitea/modules/migration"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseGithubExportedData(t *testing.T) {
```

Comment:
Just a FYI, the CI errors are related, seems like the test values needs a bit of tweaking.